addons/web_editor/static/src/js/backend/field_html.js

```diff
@@ -183,7 +183,7 @@ var FieldHtml = basic_fields.DebouncedField.extend(TranslatableFieldMixin, {
             recordInfo: {
                 context: this.record.getContext(this.recordParams),
                 res_model: this.model,
-                res_id: this.res_id,
+                res_id: this.res_id | this.record.getContext(this.recordParams).default_res_id,
```

Comment:
`||` ?

Reply:
yes...of course, I was in the lucky case in which this.red_id was undefined, but the bitwise operator is not really suitable here